src/main/default/lwc/createCaseButton/createCaseButton.js

```diff
@@ -101,6 +103,12 @@ export default class CreateCaseButton extends LightningElement {
       this.loading = true;
       const newCaseId = await newCase(this.caseData);
       if (newCaseId) {
+        this.engine.dispatch(
+          this.actions.updateCaseInput({
+            fieldName: 'id',
+            fieldValue: newCaseId
+          })
+        );
         this.engine.dispatch(this.actions.logCreateCase(newCaseId));
```

Comment:
If we're passing the case Id already why isn't it being sent with the event data?

Reply:
I guess I can remove it, we're passing the case id, but the actual headless action takes no payload xD